src/services/legal-s.ts

```diff
@@ -0,0 +1,32 @@
+/** A legal filing object from the API. */
+export interface LegalFilingI {
+  // contains one arbitrarily-named legal filing output
+  // whose name is the filing type
+  // eg, alteration, annualReport, etc
+  [name: string]: string // link to fetch it
+}
+
+/** Response object from LegalServices.fetchDocuments(). */
+export interface FetchDocumentsI {
+  // contains any number of arbitrarily-named submission level outputs
+  // or an array named "legalFilings"
+  // eg, certificate, noticeOfArticles, receipt, etc
+  [name: string]: string | LegalFilingI[] // link to fetch it, or array
+}
+
+/**
+ * Fetches documents object.
+ * @param url the full URL to fetch the documents
+ * @returns the fetch documents object
+ */
+export const fetchDocuments = async (url: string): Promise<Blob> => {
```

Comment:
Why the 'services' folder? Could we put it in utils or maybe composable? Seems like we could use this across many pdf scenarios

Reply:
moved it in download utils !